sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/SystemEventMappingNames.java

```diff
@@ -0,0 +1,816 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file should be out of implementation

Reply:
This will be moved to the root package as a follow-up PR when we reconcile the generated system event constant names with the handwritten one.